home/src/main/resources/rdf/display/firsttime/search_individuals_vitro.n3

```diff
@@ -35,23 +35,49 @@
 :sort_title_desc  a                vitro-search:Sort ;
         vitro-search:order         20 ;
         vitro-search:sortField     :field_label_sort ;
+        vitro-search:hasFallback   :sort_name_raw_desc ;
         vitro-search:isAscending   false ;
+        vitro-search:display       true ;
         vitro-search:id            "titledesc" .
 
-:sort_title_asc  a                vitro-search:Sort ;
+:sort_title_asc  a                 vitro-search:Sort ;
         vitro-search:sortField     :field_label_sort ;
         vitro-search:isAscending   true ;
+        vitro-search:hasFallback   :sort_name_raw_asc ;
         vitro-search:order         30 ;
+        vitro-search:display       true ;
         vitro-search:id            "titleasc" .
 
+:sort_name_raw_desc  a             vitro-search:Sort ;
+        vitro-search:order         40 ;
+        vitro-search:sortField     :field_name_raw ;
+        vitro-search:isAscending   false ;
+        vitro-search:id            "name_raw_desc" .
+
+:sort_name_raw_asc  a                vitro-search:Sort ;
+        vitro-search:sortField     :field_name_raw ;
+        vitro-search:isAscending   true ;
+        vitro-search:order         50 ;
+        vitro-search:id            "name_raw_asc" .
```

Comment:
it would be nice if order of properties is the same in all classes

Reply:
Fixed